schemas/json/theme.json

```diff
@@ -317,6 +317,33 @@
 									"fontFamily": {
 										"description": "CSS font-family value.",
 										"type": "string"
+									},
+									"fontFace": {
+										"description": "",
+										"type": "array",
+										"items": {
+											"type": "object",
+											"fontFamily": {
+												"description": "CSS font-family value.",
+												"type": "string"
+											},
+											"fontWeight": {
+												"description": "List of available font weights, separated by a space.",
+												"type": "string"
+											},
+											"fontStyle": {
+												"description": "CSS font-style value.",
+												"type": "string"
+											},
+											"fontStretch": {
+												"description": "CSS font-stretch value.",
+												"type": "string"
+											},
+											"src": {
+												"description": "Paths or URLs to the font files.",
+												"type": "array"
```

Comment:
Would it be possible to also provide the type for the individual items in the array?